src/config.js

```diff
@@ -277,6 +277,11 @@ export function newConfig() {
             utils.logWarn(`Auction Options ${k} must be only string`);
             return false
           }
+        } else if (k === 'suppressStaleRender') {
+          if (typeof val[k] !== 'boolean') {
```

Comment:
Nitpicking a bit but I maybe consider using the utils function for type checks

Reply:
Fixed.  Thanks.